lib/vanagon/platform/solaris_10.rb

```diff
@@ -42,14 +42,16 @@ def generate_package(project)
           # - The bin directory and all bin files are owned by root:bin instead of root:sys
           # - All files under lib are owned by root:bin instead of root:sys
           # - All .so files are owned by root:bin instead of root:sys
+          # - Explicity only include puppet directories 
+          #   (this should exclude things like root/bin root/var and such)
           %((cd $(tempdir)/#{name_and_version}; pkgproto . | sort | awk ' \
-            $$1 ~ /^d$$/ {print "d",$$2,$$3,"0755 root sys";} \
+            $$1 ~ /^d$$/ && $$3 ~ /^.*puppet.*/ {print "d",$$2,$$3,"0755 root sys";} \
```

Comment:
Below on line https://github.com/puppetlabs/pe_acceptance_tests/blob/2015.3.x/acceptance/tests/file-sync/file-sync_commit.rb you can see `project.get_directories` is called. That will return a list of directories, which have a path attribute, which is probably what you care about most.